src/features/user/utils/fetch-data.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryClient } from "@tanstack/react-query";
+
```

Comment:
Nazwa pliku fetch-data o niczym nie mówi. Ten plik rozbij na 2: update-user.ts i update-image.ts

Reply:
I zrób nowy folder /api i w nim to zrób zamiast w /utils